.gitignore

```diff
@@ -127,3 +128,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# VSCode settings
+.vscode
```

Comment:
You can set the global git config `core.excludesfile` to exclude IDE files in every project.

Reply:
Had no idea that existed! Thank you!